package.json

```diff
@@ -65,7 +65,7 @@
     "merge-options": "^3.0.4"
   },
   "peerDependencies": {
-    "react-native": "^0.0.0-0 || 0.60 - 0.68 || 1000.0.0"
+    "react-native": "^0.0.0-0 || 0.60 - 0.69 || 1000.0.0"
```

Comment:
Why not we just go ahead and support any version 60+?

Reply:
At some point native modules will be deprecated, and this is a really good way to get people to upgrade to the latest version 😛

Reply:
make sense!